streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaDStreamLike.scala

```diff
@@ -50,7 +50,7 @@ trait JavaDStreamLike[T, This <: JavaDStreamLike[T, This, R], R <: JavaRDDLike[T
 
   def wrapRDD(in: RDD[T]): R
 
-  implicit def scalaIntToJavaLong(in: DStream[Long]): JavaDStream[jl.Long] = {
+  implicit def scalaLongToJavaLong(in: DStream[Long]): JavaDStream[jl.Long] = {
```

Comment:
this will actually break binary compatibility though -- maybe it's not worth fixing.

i'd add a line of comment saying it is just unfortunate that we made a mistake in naming.